### PR TITLE
68/featured poets block

### PR DESCRIPTION
**Pull request by StephanieF**
Please let me know if you need any changes

### CODE REVIEW

**Comment by dustinleblanc** [comment]
@StephanieF can you rebase this?

**Comment by StephanieF** [comment]
@dustinleblanc Sure thing, I'll have that up in just a sec

File: stories/featuredPoet.stories.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import Vue from "vue";
```
Comment (serundeputy) [inline stories/featuredPoet.stories.js]: @StephanieF these are the storybook things?  If so I think you should kill them off.

File: stories/featuredPoetsDeck.stories.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import Vue from "vue";
```
Comment (serundeputy) [inline stories/featuredPoetsDeck.stories.js]: storybook file? kill?

**Pull request merged**